Cargo.toml

```diff
@@ -7,12 +7,12 @@ build = "build.rs"
 
 [dependencies]
 libc = "*"
-rust-htslib = "0.19"
+rust-htslib = { git = "https://github.com/10XGenomics/rust-htslib.git", branch = "lh/featurize-bzip2-lzma", default-features = false, features = ["serde"] }
```

Comment:
I think all our changes should now be released in rust-htslib=0.21.0.  Let me know if anything is missing.